src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/InstrumentationType.cs

```diff
@@ -0,0 +1,23 @@
+namespace OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.Configuration
+{
+    /// <summary>
+    /// Enum representing all instrumentations.
+    /// </summary>
+    public enum InstrumentationType
```

Comment:
There is already `IntegrationIds` enum. We should unite these.

Reply:
I have added fields from InstrumentationType to IntegrationIds